### ISSUE TITLE
warning: timeout (-t) value larger than period (-p) produces unexpected results

### ISSUE THREAD

**Issue by MrDragon1122** [schweikert/fping#142]
warning: timeout (-t) value larger than period (-p) produces unexpected results

fping -r 0 -c 10 -B 1.0 -t 300 -i 0 -p 1 -u host1 host2 ... host1000

Hello, I want to consult, what will happen to the above warning?  unexpected results ？

thanks！

**Comment by MrDragon1122** [comment]
fping 4.1

**Comment by MrDragon1122** [comment]
By looking at the documentation, I found that after fping 4.X, -t represents a separate timeout for each packet, so that the settings for -t and -p are not relevant, so whether this warning can be ignored ?

**Comment by schweikert** [comment]
I agree that the warning is a bit confusing and probably not needed. I have removed it.

**Comment by bkuker** [comment]
I am just confusing the heck out of myself here. I started on issue #32
On both 
I think the "Unexpected results" is only the "incorrect" running packet loss count?

the docs also say "In loop/count mode, the default timeout is automatically adjusted to match the "period" value (but not more than 2000ms). You can still adjust the timeout value with this option, if you wish to, but note that setting a value larger than "period" produces inconsistent results, because *the timeout value can be respected only for the last ping.*"

But it looks like the timeout I specify below is being respected for all pings?

20ms period, 70ms timeout:

```
bkuker@bill-system:~/fping$ fping --size=200 --period=20 --count=10 --timeout=70 www.csh.rit.edu
fping: warning: timeout (-t) value larger than period (-p) produces unexpected results

www.csh.rit.edu : xmt/rcv/%loss = 10/0/100%
```

20ms period, 100ms timeout:

```
bkuker@bill-system:~/fping$ fping --size=200 --period=20 --count=10 --timeout=100 www.csh.rit.edu
fping: warning: timeout (-t) value larger than period (-p) produces unexpected results
www.csh.rit.edu : [0], 228 bytes, 77.5 ms (77.5 avg, 75% loss)
www.csh.rit.edu : [1], 228 bytes, 77.6 ms (77.5 avg, 60% loss)
www.csh.rit.edu : [2], 228 bytes, 77.5 ms (77.5 avg, 50% loss)
www.csh.rit.edu : [3], 228 bytes, 77.4 ms (77.5 avg, 42% loss)
www.csh.rit.edu : [4], 228 bytes, 77.4 ms (77.5 avg, 37% loss)
www.csh.rit.edu : [5], 228 bytes, 77.5 ms (77.5 avg, 33% loss)
www.csh.rit.edu : [6], 228 bytes, 77.4 ms (77.5 avg, 30% loss)
www.csh.rit.edu : [7], 228 bytes, 77.4 ms (77.5 avg, 20% loss)
www.csh.rit.edu : [8], 228 bytes, 77.6 ms (77.5 avg, 10% loss)
www.csh.rit.edu : [9], 228 bytes, 77.4 ms (77.5 avg, 0% loss)

www.csh.rit.edu : xmt/rcv/%loss = 10/10/0%, min/avg/max = 77.8/77.9/78.0
```
